src/Component.cpp

```diff
@@ -799,7 +799,7 @@ void Component::serializeComponents(FileInterface* file) {
 	}
 }
 
-void Component::shootLaser(const glm::mat4& mat, WideVector& color, float size, float life) {
+void Component::shootLaser(const glm::mat4& mat, WideVector color, float size, float life) {
```

Comment:
Can we make this a const WideVector& instead? I don't think I meant to leave out const before.

Reply:
Hmm, well that compiles, for some reason I was under the impression it wouldn't work. For reference: the original reason I ended up changing this to pass by value was because we have some rvalues going into this function. I messed around with it and got some compile issues with the Sol2 bindings before I settled on passing by value. :man_shrugging: